styles.css

```diff
@@ -5,13 +5,12 @@
 .shoe4 { height: 100px; width: 180px; }
 .shoppingcart { width: 15px; }
 h1 { color: red; font-family: 'PT Sans Narrow', sans-serif;  font-size: 10px;  }
-ul { display: inline-block; }
-li { display: inline-block; color: red;  margin: 10px; }
-
-nav { position: relative;  }
+nav { position: relative; }
 footer { color: red; font-family: 'PT Sans Narrow', sans-serif;  font-size: 10px; }
```

Comment:
This `footer` rule is what's causing your footer text to be a different font & size than you nav bar up above. I didn't change anything here, but wanted to point it out - you may want to revisit this if you want the footer & header to be more alike.